cg/services/delivery_message/messages/analysis_scout_message.py

```diff
@@ -0,0 +1,16 @@
+from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
+from cg.services.delivery_message.messages.utils import get_fastq_delivery_path, get_scout_link
+from cg.store.models import Case
+
+
+class AnalysisScoutMessage(DeliveryMessage):
+    def create_message(self, case: Case) -> str:
```

Comment:
should this be able to get mutlple scout links that can be pooled in one message. Imagine a ticket with 30 cases, that would be 30 emails or?

Reply:
This is not implemented for tickets currently, although that is up for refinement.

Reply:
Ok, but how will the current implementation work in practice? Will there be 30 messages in a ticket with 30 cases like this? Is that what we want in the end? Or maybe I am missing a step here.

Reply:
Yes, the implementation follows the acceptance criteria defined by production. They want it on a case basis